tools/init_databases.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+# Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+# For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+# import required module
+import os
+import sqlite3
+import argparse
+
+
+# this helper script creates DBs from SQL schema files
+
+def process_file(path: os.path, dst_directory: os.path) -> None:
+    file_name = os.path.basename(path)
+    print(file_name)
+
+    db_name = file_name.split("_0")[0]
+    db_name = db_name + ".db"
```

Comment:
Keep in mind that using f-strings is "more pythonic" than "+" sign concatenation & it's far more efficient when there are more strings arguments to join

Reference: https://stackoverflow.com/questions/59180574/string-concatenation-with-vs-f-string

Reply:
fixed

Reply:
Ok